src/euphorie/deployment/locales/it/LC_MESSAGES/euphorie.po

```diff
@@ -6945,27 +7350,607 @@ msgid "warn_risk_present"
+msgstr "Certificato"
+
+msgid "label_certificate_datetime_example"
+msgstr "il <time class=\"certificate-date\">7 settembre 2019</time>"
+
+msgid "label_certificate_department"
+msgstr "Dipartimento/Regione"
+
+msgid "label_complete"
+msgstr "Completo"
+
+msgid "label_youve_earned_a_certificate_for_this_tool"
+msgstr "Hai ottenuto un certificato per questo strumento"
+
+msgid "label_certificate_benefits"
+msgstr ""
+"Con questo certificato, puoi richiedere una riduzione delle tariffe "
+"assicurative della tua azienda"
+
+msgid "label_certificate_learn_more_about_the_benefits"
+msgstr "Scopri di più sui vantaggi…"
+
+msgid "label_certificate_view"
+msgstr "Visualizza certificato"
+
+msgid "label_awarded_certificates"
+msgstr "Certificati assegnati"
+
+msgid "label_certificate_awarded"
+msgstr "I seguenti certificati sono stati assegnati per questa valutazione."
+
+msgid "label_certificate_country"
+msgstr "paese"
+
+msgid "label_certificate_company_identification_number"
+msgstr "Numero di identificazione dell'azienda"
+
+msgid "label_certificate_contact_name"
+msgstr "Nome del contatto"
+
+msgid "label_certificate_for_conducting"
+msgstr "Per condurre una valutazione del rischio con lo strumento"
+
+msgid "label_certificate_is_private"
+msgstr ""
+"è privato a meno che non si scelga di renderlo pubblico. Se fai clic sul "
+"pulsante \"Pubblica\" qui sotto, il tuo certificato sarà pubblicato "
+"sull'ufficiale"
+
+msgid "label_certificate_presented_to"
+msgstr "Questo certificato è presentato a"
+
+msgid "label_certificate_your_certificate_for"
+msgstr "Il tuo certificato per"
+
+msgid "label_constantly"
+msgstr "Costantemente"
+
+msgid "label_continue_action_plan"
+msgstr "Vai alla sezione misure di miglioramento"
+
+msgid "label_download_report"
+msgstr ""
+"Consigliamo di tenere sempre disponibile nella tua azienda una versione "
+"aggiornata e stampata del report."
+
+msgid "label_help"
+msgstr "Aiuto"
+
+msgid "label_high"
+msgstr "Alto"
+
+msgid "label_high_severity"
+msgstr "Gravità alta (molto alta)"
+
+msgid "label_large"
+msgstr "Grande"
+
+msgid "label_last_modified"
+msgstr "Salvato"
+
+msgid "label_low"
+msgstr "Basso"
+
+msgid "label_measures_overview"
+msgstr "Panoramica delle misure"
+
+msgid "label_medium"
+msgstr "Medio"
+
+msgid "label_name_ra"
+msgstr "Nome della valutazione dei rischi"
+
+msgid "label_print"
+msgstr "Stampare"
+
+msgid "label_profiling"
+msgstr "La valutazione del rischio nella tua organizzazione"
+
+msgid "label_ra_title"
+msgstr "Nome della valutazione dei rischi"
+
+msgid "label_ra_title_extended"
+msgstr "Indicare un titolo per la valutazione dei rischi"
+
+msgid "label_regularly"
+msgstr "Regolarmente"
+
+msgid "label_risks_overview"
+msgstr "panoramica dei rischi"
+
+msgid "label_select_or_add_existing_measures"
+msgstr ""
+"Per la gestione adeguata del rischio è necessario aver attuato almeno tutte "
+"le misure obbligatorie. <strong>Seleziona o aggiungi tutte le misure "
+"attuate.</strong>"
```

Comment:
There is a tailing `</strong>` that looks suspicious.

Reply:
I believe that this was another wish by the italian partner to accentuate some text. Other countries do not have it, but I know there was a special round with IT:

![Screenshot 2025-04-02 at 08 58 00](https://github.com/user-attachments/assets/7b554faa-fb69-4793-8b9c-b18b18c02528)